api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -260,8 +260,25 @@ message Bootstrap {
   // This may be overridden on a per-cluster basis in cds_config, when
   // :ref:`dns_resolution_config <envoy_v3_api_field_config.cluster.v3.Cluster.dns_resolution_config>`
   // is specified.
+  // *dns_resolution_config* will be deprecated once
+  // :ref:'typed_dns_resolver_config <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.typed_dns_resolver_config>'
+  // is fully supported.
   core.v3.DnsResolutionConfig dns_resolution_config = 30;
 
+  // DNS resolver type configuration extension. This extension can be used to configure c-ares, apple,
+  // or any other DNS resolver types and the related parameters. When c-ares DNS is configured,
+  // :ref:'dns_resolution_config <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.dns_resolution_config>'
```

Comment:
This should be `<envoy_api_field_core.DnsResultionConfig>` instead of `<envoy_v3_api_field_config.bootstrap.v3.Bootstrap.dns_resolution_config>`

The same issue is in the other proto files.

Reply:
Yeah, that's right. Thanks!

I did some research and found the complete syntax is: :ref:`DnsResolutionConfig <envoy_v3_api_msg_config.core.v3.DnsResolutionConfig>` and made the change.

Please let me know whether this look good.